web/styles/popovers.css

```diff
@@ -568,37 +568,25 @@ ul {
 
     .user_profile_manage_own_edit_button,
     .user_profile_manage_others_edit_button {
-        width: 25px;
-        height: 14px;
-        text-align: center;
-        font-size: 18px;
-    }
-
-    .user_profile_manage_own_copy_link_button {
-        width: 20px;
-        height: 15px;
-        text-align: center;
+        text-decoration: none;
         font-size: 15px;
     }
 
-    #user_profile_manage_own_copy_link {
-        height: 25px;
-        margin-right: 8px;
+    .user_profile_manage_own_copy_link_button {
+        font-size: 19px;
```

Comment:
This follows the existing pattern of setting a `font-size` on the containing element, rather than the font-icon itself. I don't love that, but this does keep things consistent with what we've got.